be/src/util/bitmap_value.h

```diff
@@ -1888,6 +2414,17 @@ class BitmapValue {
             }
             break;
         }
+        case SET: {
+            std::vector<uint64_t> values;
+            for (auto v : _set) {
+                values.emplace_back(v);
+            }
+            std::sort(values.begin(), values.end());
```

Comment:
std::set or phmap::flat_hash_set is better for at most 32 elements?

if std::set is not too bad, we can avoid copy to array and sort.

Reply:
Yes, `std::set` is good enough when the elements' count is negligible.